ta4j-core/src/test/java/org/ta4j/core/analysis/criteria/MaximumDrawdownCriterionTest.java

```diff
@@ -96,13 +96,13 @@ public void withSimpleTrades() {
                 Order.buyAt(1,series), Order.sellAt(2,series),
                 Order.buyAt(2,series), Order.sellAt(3,series),
                 Order.buyAt(3,series), Order.sellAt(4,series));
-        assertEquals(.9d, mdd.calculate(series, tradingRecord), TestUtils.GENERAL_OFFSET);
+        assertNumEquals(.9d, mdd.calculate(series, tradingRecord));
     }
 
     @Test
     public void betterThan() {
         AnalysisCriterion criterion = new MaximumDrawdownCriterion();
-        assertTrue(criterion.betterThan(0.9, 1.5));
-        assertFalse(criterion.betterThan(1.2, 0.4));
+        assertTrue(criterion.betterThan(numOf(0.9), numOf(1.5)));
+        assertFalse(criterion.betterThan(numOf(1.2), numOf(0.4)));
     }
 }
```

Comment:
MaximumDrawdown Test looks good.  Am wondering if `double` parametered `numOf(0.9)` should be `String` parametered `numOf("0.9")`